### ISSUE TITLE
Test Report a Problem Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4915]
# Page to Test:

Report a Problem - https://openlibrary.org/contact?path=/people/bpmcneilly

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly I would like on this page.

**Comment by RayBB** [comment]
Closing as the page just has links now and not a form anymore.
